datahub-classify/src/datahub_classify/similarity_predictor.py

```diff
@@ -513,10 +501,9 @@ def check_similarity(
             table_info1, table_info2, use_embeddings, pruning_mode
         )
     except Exception as e:
-        logger.error(
+        logger.exception(
```

Comment:
this should probably be a warning/info, not an exception

Reply:
Changed to warning.